build/e2e.cloudbuild.yaml

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-timeout: 7200s
+timeout: 10800s
```

Comment:
Tests are currently taking over 2 hours to run, but I have plans to improve the performance.